src/objects/page/demo/example-with-alert.twig

```diff
@@ -0,0 +1,28 @@
+{% embed '@cloudfour/objects/page/page.twig' only %}
+  {% block alert %}
+    {% include '@cloudfour/components/alert/alert.twig' with {
+      dismissable: true,
+      floating: true,
+      icon: 'cloud-slash',
+      message: 'You appear to be offline, some content or features may be unavailable.'
```

Comment:
As you pointed out in a different PR, this text is too long for the current design. We should shorten it for this example.

```suggestion
      message: 'You appear to be offline, some content may be unavailable.'
```